portal-kernel/src/com/liferay/portal/kernel/util/GroupCapabilityProvider.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.kernel.util;
+
+import com.liferay.portal.kernel.model.Portlet;
+
+/**
+ * @author Alejandro Tardín
+ */
+public interface GroupCapabilityProvider {
```

Comment:
I find weird that we have a provider for something that we don't have (there's no GroupCapability object). Why not simply calling this GroupCapability?